crates/sui-surfer/src/lib.rs

```diff
@@ -1,35 +1,38 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-use futures::future::join_all;
 use rand::rngs::StdRng;
 use rand::seq::SliceRandom;
 use rand::{Rng, SeedableRng};
 use std::path::PathBuf;
 use std::sync::Arc;
 use std::time::Duration;
 use sui_swarm_config::genesis_config::{AccountConfig, DEFAULT_GAS_AMOUNT};
-use surf_strategy::SurfStrategy;
+use sui_types::base_types::ObjectID;
+use surf_strategy::{ExitCondition, SurfStrategy};
 use test_cluster::{TestCluster, TestClusterBuilder};
-use tokio::sync::watch;
 use tracing::info;
 
 use crate::surfer_state::SurfStatistics;
 use crate::surfer_task::SurferTask;
 
 pub mod surf_strategy;
-mod surfer_state;
+pub mod surfer_state;
 mod surfer_task;
 
 const VALIDATOR_COUNT: usize = 7;
 
 const ACCOUNT_NUM: usize = 20;
 const GAS_OBJECT_COUNT: usize = 3;
 
+type EntryFunctionFilterFn =
+    Arc<dyn Fn(&str /* module */, &str /* function */) -> bool + Send + Sync + 'static>;
```

Comment:
maybe add a short comment indicating whether true filters out the function, or false filters out the function?